### PR TITLE
Setting up your development environment

### PR DESCRIPTION
**Pull request by iamanolive**
## Related Issue
resolves issue #31 for GSSOC level2

## Description

Wrote a detailed blog post on setting up a development environment for HTML coding.

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): _______________

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: docs/html/setup-environment.md [hunk 1]
```diff
@@ -5,4 +5,152 @@ sidebar_label: Setting up environment
...
+
+### Tips for a Smooth Coding Experience
+
+1. Use Version Control: Integrate Git into your workflow to track changes and collaborate with others.
+2. Stay Organized: Keep your files organized in a logical folder structure.
+3. Learn Shortcuts: Familiarize yourself with keyboard shortcuts to boost your productivity.
+
+## In Conclusion
+
+Setting up your development environment for HTML coding is a crucial step towards becoming a proficient web developer. Whether you choose a powerful IDE like Visual Studio Code, a lightweight editor like Sublime Text, or a simple tool like Notepad++, the key is to find what works best for you and your workflow.<br>
```
Comment (Ajay-Dhangar) [inline docs/html/setup-environment.md:152]: remove `<br>`

File: docs/html/setup-environment.md [hunk 2]
```diff
@@ -5,4 +5,152 @@ sidebar_label: Setting up environment
...
+
+For most users, especially beginners, Visual Studio Code is an excellent choice due to its extensive features, flexibility, and community support. It strikes a great balance between power and usability, making it a favorite among developers of all skill levels.<br>
```
Comment (Ajay-Dhangar) [inline docs/html/setup-environment.md:154]: remove `<br>` or you want to use then try `<br />`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Now update code based on suggestions

**Comment by iamanolive** [comment]
made the changes @Ajay-Dhangar anything else?

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**